kalite/distributed/features/steps/generic.py

```diff
@@ -1,6 +1,9 @@
 from behave import *
 from kalite.testing.behave_helpers import *
+from django.contrib.auth.models import User
 
 @given("I am on the homepage")
 def step_impl(context):
-    go_to_homepage(context)
\ No newline at end of file
+	if not User.objects.exists():
```

Comment:
I understand what you're saying, but my contention is that adding unexpected side effects to an existing step violates the principle of least surprise. Imagine that a breaking change is made down the road, and someone discovers that the `I am on the homepage` step fails not because they failed to be on the homepage, but because a superuser wasn't created. Therefore I would indeed like to manually update all the affected tests, since it explicitly spells out the fact that many existing features *now* depend on the fact that a superuser exists.